scripts/superdesk-monitoring/aggregate.js

```diff
@@ -381,6 +381,13 @@
             });
         };
 
+        this.reload = function() {
+            $timeout(function() {
+                self.search(self.searchQuery);
+            }, 1, false);
+            self.search();
```

Comment:
@gbbr, ideally it should just refresh current search by only calling it once. But because of the way we have set it up, search sets watched values which eventually update actual view. In its current state it's much more easier to have it working this way than refactoring, though alternative solutions are welcome.